config/src/main/resources/profiles/selfish-staker.toml

```diff
@@ -0,0 +1,6 @@
+sync-mode="X_CHECKPOINT"
+data-storage-format="BONSAI"
+bonsai-historical-block-limit=128
+max-peers=25
+Xp2p-peer-lower-bound=10
```

Comment:
I would not use the lower bound. I checked yesterday, and it only seems to be used in one place. I would like to get rid of that config option ...